School/urls.py

```diff
@@ -23,7 +23,7 @@
                     add_comment, room_detail, lesson_detail,
                     student_detail, lesson_detail_edit, student_detail_edit,
                     teacher_detail_edit, group_detail_edit, all_students, all_teachers, all_groups,
-                    registration)
+                    registration, verification)
```

Comment:
Can you change `from views import ...` to `import views`
And then use
`views.verification, views_sutdent_detail` etc ?